nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/utils/AzureStorageUtils.java

```diff
@@ -215,6 +258,67 @@ public static void validateProxySpec(ValidationContext context, Collection<Valid
         ProxyConfiguration.validateProxySpec(context, results, PROXY_SPECS);
     }
 
+    public static String validateFileSystemProperty(PropertyDescriptor propertyDescriptor, PropertyContext context, Map<String, String> attributeMap) {
+        final String fileSystem = getPropertyFromAttributeMap(propertyDescriptor, context, attributeMap);
+        return doValidateFileSystemValue(propertyDescriptor, fileSystem);
+    }
+
+    public static String validateFileSystemProperty(PropertyDescriptor propertyDescriptor, PropertyContext context, FlowFile flowFile) {
+        final String fileSystem = getPropertyFromFlowFile(propertyDescriptor, context, flowFile);
+        return doValidateFileSystemValue(propertyDescriptor, fileSystem);
+    }
```

Comment:
In my opinion, the following would be less verbose in these methods. `getPropertyFromFlowFile()` is also not needed in this case.
```suggestion
    public static String validateFileSystemProperty(PropertyDescriptor propertyDescriptor, PropertyContext context, FlowFile flowFile) {
        return validateFileSystemProperty(propertyDescriptor, context, flowFile.getAttributes());
    }
```